src/amo/components/Category/index.js

```diff
@@ -67,11 +68,17 @@ export function mapStateToProps(state, ownProps) {
     { ...state.search.filters, page: parsePage(state.search.page) },
     { ...filters, page: queryParams.page },
   );
+
+  const hasFetched = state.categories.hasFetched;
+  const loading = state.categories.loading || state.search.loading;
+
   if (filtersMatchState) {
     return {
       addonType: filters.addonType,
       category,
       filters,
+      hasFetched,
+      loading,
```

Comment:
This prop is not affecting anything because `...state.search` (below) looks to be overriding it.